core/templates/dev/head/domain/skill/SkillObjectFactory.js

```diff
@@ -119,6 +119,22 @@ oppia.factory('SkillObjectFactory', [
       return this._description;
     };
 
+    Skill.prototype.setSupersedingSkillId = function(skillId) {
```

Comment:
I thought we removed the setters?

Reply:
Sorry, looks like the merge conflicts were not all removed. It should be clean now